developer-docs-site/docs/nodes/staking.md

```diff
@@ -0,0 +1,236 @@
+---
+title: "Staking"
+slug: "staking"
+---
+import ThemedImage from '@theme/ThemedImage';
+import useBaseUrl from '@docusaurus/useBaseUrl';
+
+# Staking
```

Comment:
this doc covers a lot, it would benefit from a really high overview of what it will cover:

* Basics of staking
* Requirements and rewards for staking
* Roles and identities available in this space